### ISSUE TITLE
`del` does not work

### ISSUE THREAD

**Issue by mtarnovan** [roryqueue/redets#3]
```
pry(18)> Redix.command(redix_conn, ~w(SET FOO 1))
{:ok, "OK"}
pry(19)> Redix.command(redix_conn, ~w(DEL FOO))
{:ok, 1}
pry(20)> Redix.command(redix_conn, ~w(DEL FOO))
{:ok, 0}
pry(21)> FakeRedis.command(conn, ~w(SET FOO 1))
{:ok, "OK"}
pry(22)> FakeRedis.command(conn, ~w(DEL FOO))  
** (FunctionClauseError) no function clause matching in FakeRedis.del/3    
    
    The following arguments were given to FakeRedis.del/3:
    
        # 1
        :AHzdxN2w_fakeredis
    
        # 2
        ["FOO"]
    
        # 3
        0
```
There are also no tests for `del`.
I'll try to send a PR

**Comment by roryqueue** [comment]
thanks @mtarnovan , will take a look (and try to update this package generally) tonight and/or tomorrow

**Comment by mtarnovan** [comment]
Thanks. Would be great if you could also add [`SCAN`](https://redis.io/commands/scan), but providing the full interface incl. `COUNT` and `MATCH` may prove tricky.

**Comment by roryqueue** [comment]
the bug should be fixed in 0.2.1, lmk if not. sloppy untested typo from old bad programmer me 🤷 . added SCAN issue here: https://github.com/roryqueue/fakeredis/issues/4 , will look at that in depth at some point soon. please reopen if you find a problem persists with DEL.
